docs/decisions/0002-react-i18n.rst

```diff
@@ -0,0 +1,70 @@
+2. ``react-intl``
+-----------------
+
+Status
+------
+
+Accepted
+
+Context
+-------
+
+We have business needs to translate or otherwise internationalize
+strings of text used within ``paragon``
+components. Internationalization is critical to the goal of extending
+access to education globally, a core aspect of the mission of the
+``edx`` platform.
```

Comment:
I love the suggestion feature! Not going to try to use it's "apply suggestion" dealio, since I want to keep my commit history clean, but I love how clear this is, especially for stylistic concerns!